### PR TITLE
Save product type into project.details

### PR DESCRIPTION
**Pull request by ThomasKranitsas**
Related issue #672

### CODE REVIEW

File: src/projects/create/components/ProjectWizard.jsx [hunk 1]
```diff
@@ -43,7 +43,8 @@ class ProjectWizard extends Component {
         devices: [],
         utm: {
           code: s.projectRef
-        }
+        },
+        products: [config[s.projectType].subtypes[s.projectSubType].id]
```
Comment (vikasrohit) [inline src/projects/create/components/ProjectWizard.jsx]: https://github.com/ThomasKranitsas/connect-app/blob/ef1b699e44f95b87d264392dd2376d8f48e93dd7/src/projects/create/components/ProjectWizard.jsx#L51 should save config[projectType].id (you need to add id field to the project type objects in config json) directly not the product (subType).

Comment (ThomasKranitsas) [inline src/projects/create/components/ProjectWizard.jsx]: @vikasrohit sorry.. I'm not sure I understand what you mean here..
1) What ID(s) should I add in project types in config.json?

2) So the `project.type` should be the `Project type ID` and the `project.details.products[0]` should be the `Project subType ID`?

Comment (vikasrohit) [inline src/projects/create/components/ProjectWizard.jsx]: 1. Id of project type would be values which we were storing in type field of the project before all these changes i.e. visual_design for Design, visual_prototype for Prototype, app_dev for Development and generic for work projects.
2. Yes, correct as said above project type id is which we were storing as project type earlier.

Comment (ThomasKranitsas) [inline src/projects/create/components/ProjectWizard.jsx]: Thanks I'll push another commit in few minutes

**Comment by ThomasKranitsas** [comment]
@vikasrohit can you check the new commit? thanks!

**Comment by vikasrohit** [comment]
@gondzo although these changes should work, still please double check by running the application with these changes and merge after it.

**Pull request merged**